### ISSUE TITLE
UndefVar in extendvector `siz1`.

### ISSUE THREAD

**Issue by KristofferC** [eschnett/SIMD.jl#3]
Repo:

```jl
julia> v = Vec{4,Float64}((1,2,3,4))
       sum(v)
10.0

julia> v = Vec{3,Float64}((1,2,3))
       sum(v)
ERROR: UndefVarError: siz1 not defined
 in extendvector(::ASCIIString, ::Int64, ::ASCIIString, ::Int64, ::Int64, ::ASCIIString, ::ASCIIString) at /home/kristoffer/.julia/v0.5/SIMD/src/SIMD.jl:379
 in (::Vararg{Any})() at /home/kristoffer/.julia/v0.5/SIMD/src/SIMD.jl:1100
 in sum(::SIMD.Vec{3,Float64}) at /home/kristoffer/.julia/v0.5/SIMD/src/SIMD.jl:1136
 in eval(::Module, ::Any) at ./boot.jl:267
```

**Comment by eschnett** [comment]
Corrected via 8fe328e.

**Comment by KristofferC** [comment]
Thanks!
